src/tasks/mod.rs

```diff
@@ -310,6 +310,70 @@ pub enum GlobalEvent {
 }
 impl PassType for GlobalEvent {}
 
+pub fn quorum_filter<
+    TYPES: NodeType<ConsensusType = SequencingConsensus>,
+    I: NodeImplementation<
+        TYPES,
+        Leaf = SequencingLeaf<TYPES>,
+        ConsensusMessage = SequencingMessage<TYPES, I>,
+    >,
+>(
+    event: &SequencingHotShotEvent<TYPES, I>,
+) -> bool {
+    match event {
+        SequencingHotShotEvent::QuorumProposalSend(_, _)
+        | SequencingHotShotEvent::QuorumVoteSend(_)
+        | SequencingHotShotEvent::SendDABlockData(_)
+        | SequencingHotShotEvent::Shutdown
+        | SequencingHotShotEvent::ViewChange(_)=> true,
+
+
+        _ => false,
+    }
+}
+
+pub fn da_filter<
```

Comment:
Nit: maybe `committee_filer` is a better name, since it's for the committee networking task, not the DA task?